cli/src/trusted_command_utils.rs

```diff
@@ -34,13 +34,7 @@ pub(crate) fn get_keystore_path(trusted_args: &TrustedArgs) -> PathBuf {
 }
 
 pub(crate) fn get_identifiers(trusted_args: &TrustedArgs) -> ([u8; 32], ShardIdentifier) {
-	let mut mrenclave = [0u8; 32];
-	mrenclave.copy_from_slice(
-		&trusted_args
-			.mrenclave
-			.from_base58()
-			.expect("mrenclave has to be base58 encoded"),
-	);
+	let mrenclave = mrenclave_from_base58(&trusted_args.mrenclave);
```

Comment:
Cool, thanks :D

Reply:
This is just moved code, from @echevrier 😄